server/cmd/dcrdex/version.go

```diff
@@ -25,7 +25,7 @@ const (
 	AppName  string = "dcrdex"
 	AppMajor uint   = 0
 	AppMinor uint   = 2
-	AppPatch uint   = 0
+	AppPatch uint   = 1
```

Comment:
I don't think anything changed with the server. Is it important to keep the versions the same? But I guess release version is the same as this, which makes sense to keep the same.

Reply:
I'd probably bump them together regardless, but there was one commit for the [0.2.1 milestone](https://github.com/decred/dcrdex/milestone/13?closed=1) that touched server, the script version handling.